src/Microsoft.Health.Fhir.Import.Shared.Core/Microsoft.Health.Fhir.Import.Shared.Core.projitems

```diff
@@ -0,0 +1,15 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>F6230A0E-6457-46E7-8AB9-DF80A3387EBC</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>Microsoft.Health.Fhir.Import.Core</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)ImportCoreRegistrationExtensions.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ImportErrorSerializer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ImportResourceParser.cs" />
```

Comment:
Looks like anyway we need 3 different assembly build for specific projects. And I tried several ways but looks no one can generate different assembly with only one project and can be referenced by other project. 

For these 3 classes we have interface but the implementation still need to build on different FHIR lib. Not sure you mean we should have one assembly for import like Microsoft.Health.Fhir.Import.R4 instead of Microsoft.Health.Fhir.Import.Core and Microsoft.Health.Fhir.Import.Core.R4?